src/test/java/HiveSyncToolTest.java

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
if https://github.com/snowflakedb/snowflake-hive-metastore-connector/pull/23 is merged first, update the copyright header and package here.

Reply:
done